Snippets/scale-font.py

```diff
@@ -0,0 +1,273 @@
+    
+    # kern
+    if 'kern' in tt:
+        for i, table in enumerate(tt['kern'].kernTables):
+            for k in table.kernTable.keys():
+                table[k] = scaleValue_toUPM(table[k], UPM, target_UPM)
+  
+    # GPOS
+    if not 'GPOS' in tt: return
+    unknownAttributes = []
+    for i, lookup in enumerate(tt['GPOS'].table.LookupList.Lookup):
+        for i, subTable in enumerate(lookup.SubTable):
+            # Lookup Type 1, SinglePos
+            if lookup.LookupType == 1:
+                scaleLookupType1(subTable, UPM, target_UPM)
+            # lookup type 2: PairPos
+            if lookup.LookupType == 2:
+                scaleLookupType2(subTable, UPM, target_UPM)
+            # Lookup Type 3, CursivePos
+            if lookup.LookupType == 3:
+                scaleLookupType3(subTable, UPM, target_UPM)
+            # Lookup Type 4, MarkBasePos
+            if lookup.LookupType == 4:
+                scaleLookupType4(subTable, UPM, target_UPM)
+            # Lookup Type 5, markLigPos
+            if lookup.LookupType == 5:
+                scaleLookupType5(subTable, UPM, target_UPM)
+            # Lookup Type 6, MarkMarkPos
+            if lookup.LookupType == 6:
+                scaleLookupType6(subTable, UPM, target_UPM)
+            # Lookup Type 7, ContextPos
+            if lookup.LookupType == 7:
+                scaleLookupType7(subTable, UPM, target_UPM)
+            # Lookup Type 8, ChainContextPos
+            if lookup.LookupType == 8:
+                scaleLookupType8(subTable, UPM, target_UPM)
+            # Lookup Type 9, ExtensionPos
+            if lookup.LookupType == 9:
+                if subTable.ExtensionLookupType == 1:
+                    scaleLookupType1(subTable.ExtSubTable, UPM, target_UPM)
+                if subTable.ExtensionLookupType == 2:
+                    scaleLookupType2(subTable.ExtSubTable, UPM, target_UPM)
+                if subTable.ExtensionLookupType == 3:
+                    scaleLookupType3(subTable.ExtSubTable, UPM, target_UPM)
+                if subTable.ExtensionLookupType == 4:
+                    scaleLookupType4(subTable.ExtSubTable, UPM, target_UPM)
+                if subTable.ExtensionLookupType == 5:
+                    scaleLookupType5(subTable.ExtSubTable, UPM, target_UPM)
+                if subTable.ExtensionLookupType == 6:
+                    scaleLookupType6(subTable.ExtSubTable, UPM, target_UPM)
+                if subTable.ExtensionLookupType == 7:
+                    scaleLookupType7(subTable.ExtSubTable, UPM, target_UPM)
+                if subTable.ExtensionLookupType == 8:
+                    scaleLookupType8(subTable.ExtSubTable, UPM, target_UPM)
+
+
+def main():
+    chk = argsCheck()
+    if not chk: return
+
+    inputPath = argv[1]
+    targetUPM = int(argv[2])
+    outputPath = argv[3]
+
+    inputFont = ttLib.TTFont(inputPath)
+    scaleFont(inputFont, targetUPM)
+    inputFont.save(outputPath)
+
+def argsCheck():
+    return(argv)
```

Comment:
this code is no-op and looks incomplete. clean up or remove

Reply:
probably need more details for this specific comment, can you specify?

Reply:
ah yes ! I se what you mean… ;-)